src/app/helpers/isAmpPath.js

```diff
@@ -1,5 +1,3 @@
-const AMP_PATH_REGEX = /\/articles\/amp\/:id$/;
-
-const isAmpPath = path => AMP_PATH_REGEX.test(path);
+const isAmpPath = id => id.endsWith('.amp');
```

Comment:
This function is now not taking in a path as a variable, but only the id. It should be renamed to reflect the change.